--- A/src/base/ftlcdfil.c	2023-02-26 23:39:44.000000000 -0600
+++ B/src/base/ftlcdfil.c	2024-06-15 13:43:16.494275167 -0500
@@ -310,9 +310,10 @@
                            FT_LcdFilter  filter )
   {
     static const FT_LcdFiveTapFilter  default_weights =
-                   { 0x08, 0x4d, 0x56, 0x4d, 0x08 };
+                   { 0x1c, 0x38, 0x56, 0x38, 0x1c };
+    /* Gibson, extra weight */
     static const FT_LcdFiveTapFilter  light_weights =
-                   { 0x00, 0x55, 0x56, 0x55, 0x00 };
+                   { 0x1c, 0x38, 0x61, 0x38, 0x1c };
 
 
     if ( !library )
